SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_MaterialFarmerTools.h

```diff
@@ -0,0 +1,64 @@
+/*  Material Farmer - Happiny dust
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#ifndef PokemonAutomation_PokemonSV_MaterialFarmerTools_H
+#define PokemonAutomation_PokemonSV_MaterialFarmerTools_H
+
+#include <functional>
+#include "Common/Cpp/Options/FloatingPointOption.h"
+#include "Common/Cpp/Options/EnumDropdownOption.h"
+#include "Common/Cpp/Options/SimpleIntegerOption.h"
+#include "CommonFramework/Options/LanguageOCROption.h"
+#include "CommonFramework/Notifications/EventNotificationsTable.h"
+#include "NintendoSwitch/NintendoSwitch_SingleSwitchProgram.h"
+#include "NintendoSwitch/Options/NintendoSwitch_GoHomeWhenDoneOption.h"
+#include "PokemonSV/Options/PokemonSV_EncounterBotCommon.h"
+#include "PokemonSV/Options/PokemonSV_SandwichMakerOption.h"
+#include "PokemonSV/Programs/ShinyHunting/PokemonSV_LetsGoTools.h"
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonSV{
+
+struct MaterialFarmerOptions {
```

Comment:
If these settings will be duplicated across all material farmer programs, I'm thinking it would be better to pull this out into it's own `BatchOption`. `EncounterBotCommonOptions` is an example of this.